lib/getConfigForFile.js

```diff
@@ -4,16 +4,18 @@ const augmentConfigFull = require("./augmentConfig").augmentConfigFull
 const configurationError = require("./utils/configurationError")
 const path = require("path")
 
+/*:: type returned$config = Promise<?{ config: stylelint$config, filepath: string }>*/
+
 module.exports = function (
   stylelint/*: stylelint$internalApi*/,
   searchPath/*:: ?: string*/
-)/*: Promise<?{ config: stylelint$config, filepath: string }>*/ {
+)/*: returned$config*/ {
```

Comment:
this type is used in two places, so I've decided to move it to separate local type definition